extensions/dotenv/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "dotenv",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "1.0.0",
+  "publisher": "vscode",
+  "license": "MIT",
+  "engines": {
+    "vscode": "*"
+  },
+  "scripts": {
+    "update-grammar": "node ../node_modules/vscode-grammar-updater/bin dotenv-org/dotenv-vscode syntaxes/dotenv.tmLanguage.json ./syntaxes/dotenv.tmLanguage.json"
+  },
+  "categories": ["Programming Languages"],
+  "contributes": {
+    "languages": [
+      {
+        "id": "dotenv",
+        "extensions": [
+          ".env"
+        ],
+        "filenames": [
+          ".env",
```

Comment:
This conflicts with ini:

https://github.com/remcohaszing/vscode/blob/3cb630867018c9dec57d748c8b75b05966e33bf5/extensions/ini/package.json#L42-L45